<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tagarela</class>
 <widget class="QDialog" name="tagarela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>158</width>
    <height>124</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>158</width>
    <height>124</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>158</width>
    <height>124</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sucesso</string>
  </property>
  <widget class="QLabel" name="texto">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sucesso!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bot7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>134</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
